Energy consumption azure

ForbiddenDevil opened the pull request:
Hi @DallasBowden , please review this usecase. Thanks.

DallasBowden reviewed (changes requested):
1. Need to remove white space at the bottom for fig1.png.
2. 2024 to the Copyright notice.
3. Received this error when trying to execute the libraries inclusions:
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
Input In [4], in <cell line: 22>()
     19 from sklearn.preprocessing import MinMaxScaler
     20 import seaborn as sns
---> 22 from azureml.core import Workspace, Experiment, ScriptRunConfig
     23 from azureml.core.authentication import InteractiveLoginAuthentication
     24 from azureml.core.dataset import Dataset

ModuleNotFoundError: No module named 'azureml'

ForbiddenDevil commented:
Hi @DallasBowden ,

1. Fixed the screenshot of fig1.png
2. Updated the footer for 2024 copyright
4. The "pip install" statements install the required libraries. I had them commented out. I have uncommented them in this commit. Please try running again.

DallasBowden reviewed (approved):
Merging into production.
